tests/route/test_route_consistency.py

```diff
@@ -0,0 +1,126 @@
+import pytest
+import logging
+import re
+import time
+import math
+
+pytestmark = [
+    pytest.mark.topology('any')
```

Comment:
is this test applicable for T0/T1?

Reply:
yes Arvind.. Verified this on vms-kvm-t1-lag topology as well.